bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestTruncateTable.java

```diff
@@ -0,0 +1,131 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.hbase;
+
+import static com.google.cloud.bigtable.hbase.IntegrationTests.COLUMN_FAMILY;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.apache.hadoop.hbase.HColumnDescriptor;
+import org.apache.hadoop.hbase.HRegionLocation;
+import org.apache.hadoop.hbase.HTableDescriptor;
+import org.apache.hadoop.hbase.TableName;
+import org.apache.hadoop.hbase.client.Admin;
+import org.apache.hadoop.hbase.client.Get;
+import org.apache.hadoop.hbase.client.Put;
+import org.apache.hadoop.hbase.client.RegionLocator;
+import org.apache.hadoop.hbase.client.Table;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class TestTruncateTable extends AbstractTest {
+
+  @Test
+  public void testTruncate() throws IOException {
+    TableName tableName = IntegrationTests.newTestTableName();
+    IntegrationTests.createTable(tableName);
+    Admin admin = getConnection().getAdmin();
+    try (Table table = getConnection().getTable(tableName)) {
+      byte[] rowKey = dataHelper.randomData("testrow-");
+      byte[] qual = dataHelper.randomData("qual-");
+      byte[] value = dataHelper.randomData("value-");
+      Put put = new Put(rowKey);
+      put.addColumn(COLUMN_FAMILY, qual, 1L, value);
+      put.addColumn(COLUMN_FAMILY, qual, 2L, value);
+      table.put(put);
+      assertTrue(table.exists(new Get(rowKey)));
+      admin.disableTable(tableName);
+      admin.truncateTable(tableName, false);
+      assertTrue(admin.tableExists(tableName));
+      assertFalse(table.exists(new Get(rowKey)));
+    } finally {
+      if (admin.tableExists(tableName)) {
+        admin.disableTable(tableName);
+        admin.deleteTable(tableName);
+      }
+    }
+  }
+
+  @Test
+  public void testTruncateWithSplits() throws IOException {
+    byte[][] splits = new byte[][] {
+        Bytes.toBytes("AAA"),
+        Bytes.toBytes("BBB"),
+        Bytes.toBytes("CCC"),
+    };
+
+    TableName tableName = IntegrationTests.newTestTableName();
+    HTableDescriptor descriptor = new HTableDescriptor(tableName);
+    descriptor.addFamily(new HColumnDescriptor(COLUMN_FAMILY));
+
+    Admin admin = getConnection().getAdmin();
+    admin.createTable(descriptor, splits);
```

Comment:
We have a test for createTable with splits, but I guess it doesn't hurt to do it again.